hbase-server/src/test/java/org/apache/hadoop/hbase/mob/compactions/TestPartitionedMobCompactor.java

```diff
@@ -811,10 +811,11 @@ private void compareDelFiles(List<CompactionDelPartition> delPartitions) {
   /**
    * Creates store files.
    * @param basePath the path to create file
-   * @family the family name
-   * @qualifier the column qualifier
-   * @count the store file number
-   * @type the key type
+   * @param family the family name
+   * @param qualifier the column qualifier
+   * @param count the store file number
+   * @param type the key type
+   * @param date the date value
```

Comment:
Could you please make it a little bit more specific? What's the purpose of the parameters?

Reply:
I have added some more description for parameters. Kinldy check 3rd commit. Thanks.